docs/pages/reference/config.mdx

```diff
@@ -99,12 +99,12 @@ teleport:
     # IAM join method documentation:
     # https://goteleport.com/docs/setup/guides/joining-nodes-aws-iam/
     join_params:
-        # join_method when set to "token", is equivalent to using auth_token.
-        join_method: "token"|"ec2"|"iam"
-        # When join_method is "token", token_name is either the
+        # method when set to "token", is equivalent to using auth_token.
```

Comment:
```suggestion
        # method, when set to "token", is equivalent to using auth_token.
```
Also, should this sentence be in the paragraph that begins on line 104? It seems like it belongs there since it refers to the `token` value.